spec/spec_helper.cr

```diff
@@ -232,6 +232,16 @@ module SpecHelper
     }.to_json
   end
 
+  def mock_event_query_json_error
+    {
+      "error": {"@odata.type": {
+        "code":       "string",
+        "message":    "string",
+        "innererror": {"@odata.type": "odata.error"},
+      }},
+    }.to_json
```

Comment:
This looks to be the schema, rather than example object.

There's one on the [MS docs](https://docs.microsoft.com/en-us/graph/errors#json-representation) that may be of use:
```
{
  "error": {
    "code": "invalidRange",
    "message": "Uploaded fragment overlaps with existing data.",
    "innerError": {
      "requestId": "request-id",
      "date": "date-time"
    }
  }
}
```